.github/workflows/CI.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
       - uses: actions/setup-node@v2
         with:
-          node-version: '14.19.1'
```

Comment:
There was discussion about the version in 

https://helsinkicity.slack.com/archives/C01RNQ5PV5Z/p1702026731704299

Might be best to upgrade to node 18 or even 20.

Ps. this should match the version in .nvmrc?

Reply:
It's now 16.20.2 until the "dependency can of worms" is handled in other task.